microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -1,69 +1,91 @@
+![Long-Running Task indicators for events on the main thread with slow performance](./selector-stats-images/long-running-task.png)
 
-In your performance recordings, some of these long-running tasks may be **Recalculate Style** events.  These events track the time it takes for the browser to iterate through DOM elements on a page, find all of the CSS style rules that match a given element, and then compute the element's actual style based on these rules.  These styles need to be recalculated whenever the applicability of CSS rules may have changed, such as when:
+In your performance recordings, some of these long-running tasks may be **Recalculate Style** events.  A **Recalculate Style** event tracks the time it takes for the browser to do the following:
+* Iterate through the DOM elements on a page, to find all of the CSS style rules that match a given element.
+* Compute each element's actual style, based on the matching CSS style rules.
 
-- Elements are added to or removed from the DOM.
-- An element's attributes are changed, such as `class` or `id`.
-- User input occurs, such as a mouse move or focus change, which can affect `:hover` rules.
+CSS styles need to be recalculated whenever the applicability of CSS rules may have changed, such as:
 
-When you have long-running **Recalculate Style** events, you can use the **Selector Stats** feature to understand which of your CSS selectors are taking up the most time and resulting in slow performance.
+* When elements are added to or removed from the DOM.
+* When an element's attributes are changed, such as the value of a `class` or `id` attribute.
+* When user input occurs, such as a mouse move or a change of which element in a webpage has focus, which can affect `:hover` rules.
 
-The **Selector Stats** feature provides statistics about the CSS rule selectors that were involved in one or more **Recalculate Style** events within a performance recording.
+When you have long-running **Recalculate Style** events, you can use the **Selector Stats** tab to understand which of your CSS selectors are taking up the most time and resulting in slow performance.
 
-## View CSS rule selector statistics
+The **Selector Stats** tab provides statistics about the CSS rule selectors that were involved in one or more **Recalculate Style** events within a performance recording.
 
-#### Record a performance trace with Selector Stats enabled
 
-To view the statistics of your CSS rule selectors during long-running **Recalculate Style** events, first record a performance trace with the Selector Stats feature enabled.
+<!-- ====================================================================== -->
+## Record a performance trace with Selector Stats enabled
+
+To view the statistics of your CSS rule selectors during long-running **Recalculate Style** events, first record a performance trace with the Selector Stats feature (advanced rendering instrumentation) enabled.
```

Comment:
```suggestion
To view the statistics of your CSS rule selectors during long-running **Recalculate Style** events, first record a performance trace with the Selector Stats tab (advanced rendering instrumentation) enabled.
```

Do you think we should update "Selector Stats feature" to "Selector Stats tab" throughout the article. "Tab" feels better to me.

Reply:
"tab" is best in most places.  "feature" seemed best here -- but here i wish to also mention: 
the **Enable advanced rendering instrumentation** checkbox
which you must enable.  re: std UI verbiage, you _enable_ the _checkbox_, which displays the _tab_ & enables the _feature_.

though it will be verbose, i want to expand the above sentence to be more precise & informative this way.

Reply:
```
To view the statistics of your CSS rule selectors during long-running **Recalculate Style** events, first record a performance trace with the Selector Stats feature enabled.  You enable the Selector Stats feature by selecting the **Enable advanced rendering instrumentation** checkbox, which displays the **Selector Stats** tab.
```